extensions/typescript-language-features/src/tsServer/spawner.ts

```diff
@@ -216,6 +226,13 @@ export class TypeScriptServerSpawner {
 			}
 		}
 
+		if (configuration.enableTsServerTracing && !isWeb()) {
```

Comment:
Does this need a version check?

Reply:
I thought the server would silently ignore unsupported arguments.